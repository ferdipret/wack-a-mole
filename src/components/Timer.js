import React from "react"
import styled from "styled-components"
import PropTypes from "prop-types"

const TimeWrapper = styled.div`
  text-align: right;
`

function Timer({ timer }) {
  return <TimeWrapper>Time remaining: {timer}</TimeWrapper>
}

Timer.propTypes = {
  timer: PropTypes.number.isRequired,
}

export default Timer
