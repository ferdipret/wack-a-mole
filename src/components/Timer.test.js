import React from "react"
import { shallow } from "enzyme"
import Timer from "./Timer"

describe("Timer", () => {
  const props = {
    timer: 60,
  }
  const wrapper = shallow(<Timer {...props} />)

  it("renders properly", () => {
    expect(wrapper).toMatchSnapshot()
  })
})
