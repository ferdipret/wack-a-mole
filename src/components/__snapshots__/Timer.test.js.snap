// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Timer renders properly 1`] = `
<styled.div>
  Time remaining: 
  60
</styled.div>
`;
